### PR TITLE
Milestone 3: feature styling

### PR DESCRIPTION
**Pull request by dansantander**
**This Pull Request complies with the following requirements:**

1. Use flex display mode for App, Display and ButtonPanel components

- The Calculator width should be 700px.

2. Style the Display component

- It should have a gray background.
- It should have a height set to 100px.
- The result should be presented with a white and bold text.
- The result should have padding from the edges.
- The result should be aligned to the right.

3. Style the ButtonPanel component

- Each row (display and group buttons) should have a height of 100 px.
- The Button groups should be displayed horizontally in rows.

4. Style the Button component

- Button should take 25% of the container width, except for the button that represents the “0” (zero).
- The text in the buttons should be centered and displayed in black color.
- Each button should have a border.

5. Pass “color” and “wide” props to the Button

- The color prop should be used to customize the color of the button. If it is not passed, the button should have an orange background by default.
- The wide prop should accept a boolean to indicate that the button should have twice the standard width (used for the “0” button).

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
-function Button({ name }) {
+function Button({ name, color, wide }) {
+  const style = {
+    backgroundColor: color,
+    width: wide ? '50%' : '25%',
+  };
+
   return (
-    <button type="submit">{ name }</button>
+    <button type="button" className="button" style={style}>{ name }</button>
```
Comment (jofer86) [inline src/components/Button.js:11]: Let's avoid using inline styling, we could use the orange prop to make reference to a class instead, that way we can avoid the inline styles.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -3,34 +3,34 @@ import Button from './Button';
...
+        <Button name="1" color="#e0e0e0" />
+        <Button name="2" color="#e0e0e0" />
+        <Button name="3" color="#e0e0e0" />
         <Button name="+" />
       </div>
-      <div>
-        <Button name="0" />
-        <Button name="." />
+      <div className="buttonGroup">
+        <Button name="0" color="#e0e0e0" wide="true" />
```
Comment (jofer86) [inline src/components/ButtonPanel.js]: Wide prop should be a boolean, not a string. As stated here 
image

**Review by jofer86 — CHANGES_REQUESTED** [review]
## `const STATUS = "CHANGES REQUIRED"`⚠️ ⚠️ ⚠️ 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- The project looks really clean and sharp, congrats on your artist eye!!!
- There are still some aspects that need a little bit of work, first let's start with the calculator component, as stated here: 
image
Needs to have a flex display right now is displayed as block 
image
Please do make the necessary adjustments for that.
- There are more observations down below on your code, please do tend to those as well.
- Once you are done please submit for a re-review, not much to change, all in all great job!! keep it up!
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Review by bellom — APPROVED** [review]
## Hi @dansantander  :clap

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**